### PR TITLE
Changed minimum version to 7.6 and fixed 8.7 problems.

### PR DESCRIPTION
**Pull request by cvisys**
Changed minimum version to 7.6 and removal old code.
Fixed problems with removed functions in 8.7.
Fixed problem with new namespace newer versions of solrfal.

### CODE REVIEW

File: Classes/Security/CheckPermissions.php [hunk 1]
```diff
@@ -133,10 +133,10 @@ public function checkFolderRootLineAccess(Folder $folder, $userFeGroups)
     /**
      * Get permissions set on folder (no root line check)
      *
-     * @param FolderInterface $folder
```
Comment (frans-beech-it) [inline Classes/Security/CheckPermissions.php]: please keep the interface

File: Configuration/TCA/tx_falsecuredownload_folder.php [hunk 2]
```diff
@@ -14,7 +14,7 @@
             'ignoreWebMountRestriction' => true,
             'ignoreRootLevelRestriction' => true,
         ),
-        'iconfile' => \TYPO3\CMS\Core\Utility\ExtensionManagementUtility::extRelPath('fal_securedownload') . 'Resources/Public/Icons/folder.png'
+        'iconfile' => \TYPO3\CMS\Core\Utility\ExtensionManagementUtility::siteRelPath('fal_securedownload') . 'Resources/Public/Icons/folder.png'
```
Comment (frans-beech-it) [inline Configuration/TCA/tx_falsecuredownload_folder.php]: `'iconfile' => 'EXT:fal_securedownload/Resources/...`

**Comment by frans-beech-it** [comment]
Tnx for the PR :+1:

**Comment by cvisys** [comment]
No problem.

**Pull request merged**